### PR TITLE
Fix multi-line code blocks

### PR DESCRIPTION
**Pull request by kevinlin1**
#922. Preview still renders incorrectly since markdown.js doesn't support Github-flavored markdown.

screenshot 2016-10-19 at 6 04 34 pm

### CODE REVIEW

**Comment by Sumukh** [comment]
It look like this change introduces an XSS vuln (see circleci)

File: server/templates/diff.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
       </div>
       {% endif %}
     </div>
-    <div class="comment-body markdown-body">{{ comment.message | markdown }}</div>
+    <div class="comment-body markdown-body">{{ comment.message | safe | markdown }}</div>
```
Comment (kevinlin1) [inline server/templates/diff.html]: This seems to cause the problem but at the cost of breaking valid characters like `'`.

**Comment by kevinlin1** [comment]
image

**Comment by knrafto** [comment]
I think I fixed all the double-escaping issues. I made another issue for the JS library (#926)

**Comment by knrafto** [comment]
LGTM, merging

**Pull request merged**